.github/workflows/e2e-tests.yml

```diff
@@ -82,6 +80,17 @@ jobs:
           composer require --dev --update-with-all-dependencies "nunomaduro/larastan:*"
 
       - name: "Perform static analysis"
-        run: |
-          cd e2e/
-          composer exec phpstan
+        working-directory: e2e
+        run: composer exec phpstan analyse -- -c "../larastan/e2e/${{ matrix.config }}"
+
+      - name: "Generate baseline"
+        if: ${{ failure() }}
+        working-directory: e2e
+        run: composer exec phpstan analyse -- -c ../larastan/e2e/${{ matrix.config }} -b ../larastan/e2e/${{ matrix.baseline }}
```

Comment:
Wouldn't this make the check always green? 🤔  For example I wouldn't be able to see if a PR has broke something in this analyzed repository.

If that is the case I think we can remove this part (also the uploading). And I can manually update the baseline from time to time.

Reply:
Let me test it real quick

Reply:
> And I can manually update the baseline from time to time.

I hope it will be "remove ignores from baseline"!

Reply:
This is how it is configured for the phpstan e2e test so maybe =)
But there has an issue with ignored errors. If an error is ignored in 3rd party repo (e.g. [monicahq/chandler](https://github.com/monicahq/chandler/blob/f57c49400dfa419e78bad240f018bce2949dcac2/phpstan.neon#L41)) and it is fixed by larastan the only way how we can ignore outdated errors is by completely disabling this check for repo (for example in [larastan](https://github.com/nunomaduro/larastan/blob/f8afd8fc55e5fd88b406827c9155c8c6d4f5fb02/e2e/monicahq-chandler.neon#L5 ) and in [phpstan](https://github.com/phpstan/phpstan/blob/af751d33d0dab6f257e7dc97f75cd371a3466562/e2e/integration/drupal.neon#L7))

Reply:
Ok, I got it now. Thank you 👍🏽